client/src/app/modules/admin/modules/products/pages/edit-product/edit-product.component.html

```diff
@@ -0,0 +1,381 @@
+<div class="grid">
+  <div class="col-8">
+    <div class="card">
+      <form
```

Comment:
Would be great to have one form for both create and edit to not to duplicate code